charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -231,6 +231,16 @@ data:
                       container.args = container.command + container.args
                   container.command = ["tini", "--", "/entrypoint"]
                   pod.spec.containers[0] = container
+      {{ if .Values.houston.loggingSidecar.enabled }}
```

Comment:
some more comments would be useful. E.g. what is curl -fsSL doing

Reply:
```suggestion
                  # For sidecar logging we override the default entrypoint to redirect
                  # logs to files that are consumed by the sidecar container. The command
                  # ends with a POST to localhost:8000/quitquitquit, which signals the
                  # sidecar log consumer to exit.
                  log_cmd = "1> >( tee -a /var/log/{{ .Values.houston.loggingSidecar.name }}/out.log ) 2> >( tee -a /var/log/{{ .Values.houston.loggingSidecar.name }}/err.log >&2 )"
```

Reply:
Suggestion looks good @danielhoherd. Added